_site/assets/scripts/script.js

```diff
@@ -120,45 +114,48 @@ const implex = (function () {
   function onSubmit(e) {
     e.preventDefault();
 
+    const errorBorder = 'error-border';
+    const hidden = 'hidden';
```

Comment:
Bad variable names as well. Those are not `hidden` and `errorBorder`, those are  `hiddenClassName` and `errorBorderClassName`